modules/remove_map/init.sqf

```diff
@@ -1,25 +1,7 @@
 ["Remove Maps", "Removes maps and compasses from specific gear cases.", "TinfoilHate"] call FUNC(RegisterModule);
 
-	[{time > 1}, {
-		#include "settings.sqf"
+if (hasInterface) then { // As this module is client-side only, only continue if this is a client
```

Comment:
clientInit for client only inits 
https://github.com/Global-Conflicts-ArmA/Olsen-Framework-Arma-3/blob/cb7b3f72478395034a3b1a6cba27502e54237525/core/description.hpp#L53

Reply:
If you are not switching to a statement for !hasinterface you can simply exitwith on that condition in that file

Reply:
Fixed.